Add OGCL71.dat, OGRE442 and update OG656, OGRE441

Phosphorus551 opened the pull request:
Revised my previous OG656 contribution and completed a new Class 71 replacement.

LeftofZen commented on objects/Vehicle/Train/656/656 V2.blend:
remove the `V2` please, it isn't necessary

Phosphorus551 replied:
"v2" removed from file name, along with redundant old .blend file from previous version

Phosphorus551 commented:
Apologies for the messy commits. Still not the most savvy with github's webapp. 

Anyway, I fixed an issue present with the Class 71 replacement, involving the pitch of the bogies. Additionally, I updated my previous Re 4/4 I contribution with new sprites and included a new Re 4/4 II replacement. I updated the title of the PR to reflect these additional contributions.

<img width="738" height="335" alt="NewLocos" src="https://github.com/user-attachments/assets/3ba97665-e7e9-42d4-8088-9ed56ff2f341" />

LeftofZen reviewed (changes requested):
please put one vehicle per pull request. you generally also don't create a PR from your `main`, generally you make a branch in your repository, do your work, then make a pull request for that branch
